docker-compose.yaml

```diff
@@ -45,7 +48,7 @@ services:
       MAX_USERNAME_LENGTH: "$MAX_USERNAME_LENGTH"
       WEBEX_AUTHORIZATION_URL: $WEBEX_AUTHORIZATION_URL
       WEBEX_GLOBAL_SPACE_ID: $WEBEX_GLOBAL_SPACE_ID
-    command: yarn run start
+    command: yarn run serve
```

Comment:
I would not commit this and investigate later why it does not working

Reply:
Push reverted!